### ISSUE TITLE
 Simple-Calendar - no successful export ?!

### ISSUE THREAD

**Issue by Antichton** [SimpleMobileTools/Simple-Calendar#1671]
Hello, I use the current version of the wonderful calendar. Unfortunately, I found that the calendar entries can not be exported, or afterwards imported again successfully on another device. 
Allegedly no entries were found despite many entries ?!
It would be great if this problem could be solved, because otherwise the painstakingly created entries cannot be transferred to another device.

Greeting

**Comment by tibbi** [comment]
What current version you mean? Tell the number at the bottom of the About page. If you check the file with some file manager, does it show 0B size?

**Comment by tibbi** [comment]
dont you have a rooted device or one modified in a similar way? Maybe some system services disabled?

**Comment by Antichton** [comment]
yes, of course the device is rooted. I also use a custom ROM. Which services do you mean ? All other communication programs work fine.
I will check everything again and then let you know ...

Have a nice evening
-- 
 Gesendet mit Tutanota, der sicheren & werbefreien Mailbox. 



15. März 2022, 23:03 von ***@***.***:

**Comment by tibbi** [comment]
Guess you disabled some system services then, but it is hard to fix such device specific issues.

**Comment by Antichton** [comment]
Hello Tibor,

I understand that.
However, I will give you the information that I found out:

Da Prorgamm does not work on AICP, nor on Lineage. (I assume that you are familiar with both of them. Both are used by thousands of users).
I also tested your app on a recent lineage without root. Unfortunately, the same problem. Always 0 bytes.

Too bad it is the best calendar for me, as well as all your other apps.
Thanks for the valuable work.

All the best in these confused times.

-- 
 Gesendet mit Tutanota, der sicheren & werbefreien Mailbox. 



17. März 2022, 23:13 von ***@***.***:
